docs/content/docs/flink/flink-getting-started.md

```diff
@@ -22,31 +22,31 @@ url: flink
 
 # Flink
 
-Apache Iceberg supports both [Apache Flink](https://flink.apache.org/)'s DataStream API and Table API. Currently,
-Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13, and 1.14. Previous versions of Iceberg also support Flink 1.11.
+Apache Iceberg supports both [Apache Flink](https://flink.apache.org/)'s DataStream API and Table API/SQL. Currently,
+Iceberg can with work Apache Flink versions 1.12, 1.13, and 1.14. Previous versions of Iceberg also support Flink 1.11.
 
 | Feature support                                             | Flink  | Notes                                                        |
 | ----------------------------------------------------------- | -----  | ------------------------------------------------------------ |
 | [SQL create catalog](#creating-catalogs-and-using-catalogs) | ✔️     |                                                              |
 | [SQL create database](#create-database)                     | ✔️     |                                                              |
 | [SQL create table](#create-table)                           | ✔️     |                                                              |
 | [SQL create table like](#create-table-like)                 | ✔️     |                                                              |
-| [SQL alter table](#alter-table)                             | ✔️     | Only support altering table properties, column and partition changes are not supported |
+| [SQL alter table](#alter-table)                             | ✔️     | Only support altering table properties. Changing column and partition are not supported. |
```

Comment:
There is no need for punctuation here and this rephrase isn't making the point more clear. Can you please revert this change?